utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/extractor/JsonToNuStruct.scala

```diff
@@ -26,13 +24,19 @@ object JsonToNuStruct {
 
     def addPath(next: String): String = if (path.isEmpty) next else s"$path.$next"
 
-    def extractObject(elementType: Map[PropertyName, SwaggerTyped]): AnyRef =
+    def extractObject(obj: SwaggerObject): AnyRef =
       extract[JsonObject](
         _.asObject,
         jo => TypedMap(
           jo.toMap.collect {
-            case (key, value) if elementType.contains(key) =>
-              key -> JsonToNuStruct(value, elementType(key), addPath(key))
+            case (key, value) if obj.elementType.contains(key) =>
+              key -> JsonToNuStruct(value, obj.elementType(key), addPath(key))
+            case (key, value) => obj.additionalProperties match {
```

Comment:
shouldnt' it be 
```
case (key, value) if obj.additionalProperties != AdditionalPropertiesBoolean(false) =>
```
so we do not have match error inside?
?

Reply:
Hmm, right!